frontend/src/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,67 @@
+import {useEffect, useState} from 'react'
+import { format, startOfMonth, endOfMonth, eachDayOfInterval, getDay, addMonths, subMonths, differenceInDays, getDaysInMonth } from 'date-fns'
+import { ArrowLeftIcon, ArrowRightIcon } from "@heroicons/react/24/outline"
+import classes from './Calendar.module.css'
+import CalendarCell from '../CalendarCell/CalendarCell'
+import CalendarWeekdayCell from '../CalendarWeekdayCell/CalendarWeekdayCell'
+
+function Calendar() {
+    const mapDayIndex = (dayIndex:number) => {
+        return (dayIndex === 0 ? 6 : dayIndex - 1);
+    }
+    
+    const addMonth = (currentDate:Date) => {
+        setCurrentDate(addMonths(currentDate, 1));
+    }
+
+    const subMonth = (currentDate:Date) => {
+        console.log(subMonth);
+        setCurrentDate(subMonths(currentDate, 1));
+    }
+    
+    const [currentDate, setCurrentDate] = useState(new Date())
```

Comment:
no semicolon here and in a few spots - yeah double check your formatter is running